src/locale/ur.js

```diff
@@ -69,10 +70,16 @@ export default {
     'nav.contactUs': 'ہم سے رابطہ',
     'nav.help': 'مدد اور تاثرات',
     'nav.aboutQuranProject': 'قرآن کریم ایک فلاں عامہ (عوامی بہبود کے لئے) منصوبہ ہے',
+    'nav.rightsReserved': 'جملہ حقوق محفوظ ہیں',
     'nav.mobile': 'موبائل',
     'nav.navigate': 'تشریف لے جائیں',
     'nav.legacySite': 'پرانی سائٹ',
 
+    'settings.reciters': 'قراء',
+    'settings.translations': 'ترجمہ',
+    'settings.tooltip': 'اوزاری ٹوٹکا مواد',
```

Comment:
we already have translation for setting, see setting.* group.  For tooltip key is `setting.tooltip.title`